codecs/MWV.js

```diff
@@ -45,12 +45,16 @@ Field Number:
 
 var Codec = require('../lib/NMEA0183');
 
+Math.radians = function(degrees) {
+  return degrees * Math.PI / 180;
+};
```

Comment:
That's where they should go, allowing them to be used across all codecs. @philgutowski do you want to update the rest or shall I go ahead and push a fix for all codecs moving to SI units everywhere?